qadi_simple_multi.py

```
@@ -264,34 +328,42 @@
-    
-    print(f"\n💡 Advantages:")
-    print(f"  • Real LLM-powered insights (not templates)")
-    print(f"  • Multi-perspective QADI analysis")
-    print(f"  • Progressive reasoning (each phase builds on previous)")
-    print(f"  • No timeout issues")
-    print(f"  • Much richer than single prompt approach")
+    
+    performance_items = [
+        f"⏱️  Total time: {total_time:.1f}s",
+        f"💰 Total cost: {format_llm_cost(total_cost)}",
+        f"🤖 API calls: 5 (4 phases + synthesis)",
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)